add useful tools list and add values page (#4)
koor-tech/handbook.koor.tech: docs/values.md

---
title: "Values"
---

This page describes our 3 core values including a "why" each value is important to us.

## Authenticity

Be authentic in what you are doing.

> Why should you be authentic? Every one has their own skills and ways of doing things.

Reviewer: ```suggestion
Why should you be authentic?
Everyone has their own skills and ways of doing things.
```

> 

Reviewer: ```suggestion
```

We can support our customers and ourselves best, through our individual experiences and skills.
Being honest with ourselves and our customers to build a relationship that can accept
Trust. Being clear on what we can do for each other.

## Empowerment

> Empower yourself, your colleagues and our customers by helping with your experience and knowledge.

Reviewer: ```suggestion
* Thinking in Open
  Empower yourself, colleagues and customers by thinking in the open.
```

> We need to share our experience and knowledge with each other to benefit our goal of helping our customers and the communities.

Reviewer: ```suggestion
* We need to share our experiences and knowledge openly with each other to empower ourselves in our goal of helping our customers and the communities.
```

> * Ownership of problems and situations -> Empower yourself by owning up to failure but also success, transparent to the customer.

Reviewer: ```suggestion
* Taking Ownership Of Problems And Situations
  Empower yourself by being decisive and owning up to failures and success.
```
